android/expoview/src/main/java/versioned/host/exp/exponent/modules/internal/DevMenuModule.kt

```diff
@@ -248,7 +248,7 @@ class DevMenuModule(reactContext: ReactApplicationContext, val experiencePropert
     reactApplicationContext.runOnNativeModulesQueueThread {
       val devSupportManager = getDevSupportManager()
       devSupportManager?.devSettings?.packagerConnectionSettings?.inspectorServerHost?.let {
-        val url = "http://$it/inspector?applicationId=${reactApplicationContext.packageName}"
+        val url = "http://$it/_expo/inspector?applicationId=${reactApplicationContext.packageName}"
```

Comment:
```suggestion
        val url = "http://$it/_expo/debugger?applicationId=${reactApplicationContext.packageName}"
```

The "new" name that we use in `@react-native/dev-middleware` is "debugger". Right now, it uses the endpoint `/open-debugger`, so let's keep that somewhat similar.